src/client/components/Navigation/NavGeo/SatelliteMosaic/MosaicControl/MosaicControl.tsx

```diff
@@ -38,6 +38,7 @@ const MosaicControl: React.FC = () => {
       <OptionLabel>{t('common.year')}</OptionLabel>
       <SelectPrimary
         isClearable={false}
+        maxMenuHeight={126} // 4 options with 28px height each, plus half of another option
```

Comment:
Added extra height so it's more obvious that the select has more options.
This:
![image](https://github.com/openforis/fra-platform/assets/41337901/c2f9a299-0149-473a-8c07-f621aa545772)

As opposed to this:

![image](https://github.com/openforis/fra-platform/assets/41337901/948f0b87-163c-4fa8-aff9-e09576ac28f1)